### ISSUE TITLE
Validating the service when deploying a cloud service slows down other deployments

### ISSUE THREAD

**Issue by jaanclaeys** [Microsoft/vsts-tasks#7872]
## Environment
VSTS        
Hosted and Private: 
    
## Issue Description

When using Azure Cloud Deployment powershell for cloud services, the added extra step of validation of the cloud service (introduced begin june 2018) adds two minutes per deployment. With lots of cloud services deploying in one service, this can significally slow down other tasks. Also if you work with small machines there is no way a New deployment is going to be ready in two minutes.

I would like to see an optional parameter to remove validation, because in our case we validate all the deployed services afterwards before swapping, in the meantime we can do other tasks or start our other deployments.

**Comment by Ajay-MS** [comment]
@jaanclaeys 

I agree with your feedback to make validation of cloud service as optional. We will add it soon and will update you.
